src/main/menus/app.js

```diff
@@ -136,6 +136,13 @@ var createTemplate = function(mainWindow, config) {
       click: () => {
         mainWindow.webContents.send('zoom-in', -1);
       }
+    }, {
+      label: 'Zoom Out (hidden)',
+      accelerator: 'CmdOrCtrl+_',
```

Comment:
Thanks @kytwb for your work for this pull request, much appreciated.

Do you need any help with the keyboard layouts, we'd be happy to guide you if needed.